src/main/java/net/syscon/elite/web/config/ResourceServerConfiguration.java

```diff
@@ -32,7 +32,7 @@ public void configure(final HttpSecurity http) throws Exception {
                         "/health", "/info", "/ping", "/health/ping", "/h2-console/**",
                         "/v2/api-docs", "/api/swagger.json",
                         "/swagger-ui.html", "/swagger-resources", "/swagger-resources/configuration/ui",
-                        "/swagger-resources/configuration/security").permitAll()
+                        "/swagger-resources/configuration/security", "/auth/.well-known/jwks.json").permitAll()
```

Comment:
I'm not happy about opening up the `/auth/.well-known/jwks.json` endpoint when it is only required for running locally.  Ideally I just want to permit all to this endpoint when in the `dev` profile, but I couldn't find a way to do that.